### PR TITLE
Chromium 84 upgrade

### PR DESCRIPTION
**Pull request by jstkdng**
Made some changes to the PKGBUILD as well.

### CODE REVIEW

**Comment by wchen342** [comment]
v84 is on the way so maybe wait a little longer?

**Comment by jstkdng** [comment]
Without that patch, any site that uses media breaks, like facebook, whatsapp, etc.
The issue affects 84 too, I just backported the patch.

**Comment by wchen342** [comment]
I was trying to say that just combine this PR with the update to 84. But sure we can merge this first.

**Comment by jstkdng** [comment]
Since there's no ETA, could be some time before we get 84. Better fix this before we get reports.

**Comment by jstkdng** [comment]
well, wasnt expecting 84 to come out *this* soon. I'll add the 84 changes once the OBS build is done.

**Comment by jstkdng** [comment]
Alright, that should do it. OBS build was succesful. Guess we only need @Eloston 's feedback.

File: README.md [hunk 1]
```diff
@@ -4,13 +4,22 @@ Arch Linux packaging for [ungoogled-chromium](//github.com/Eloston/ungoogled-chr
 
 ## Downloads
 
-Available in the AUR as `ungoogled-chromium`
+Available in the AUR as `ungoogled-chromium`, wayland users could benefit from using `ungoogled-chromium-ozone` and `ungoogled-chromium-git` is for the bleeding edge experience.
```
Comment (Eloston) [inline README.md:7]: Probably better to format this as a list so you can explain the differences between these packages in more detail.

File: README.md [hunk 2]
```diff
@@ -4,13 +4,22 @@ Arch Linux packaging for [ungoogled-chromium](//github.com/Eloston/ungoogled-chr
...
+### Binary Downloads
 
-Alternatively, [get builds from the Contributor Binaries website](//ungoogled-software.github.io/ungoogled-chromium-binaries/).
+You can get pre-built binaries from the following sources, just follow the instructions on each site:
 
-**Source Code**: It is recommended to use a tag. You may also use `master`, but it is for development and may not be stable.
+[Contributor Binaries website](//ungoogled-software.github.io/ungoogled-chromium-binaries/) - Binaries
+contributed by ungoogled chromium users.
+
+** Unofficial Repositories **
```
Comment (Eloston) [inline README.md:16]: Did you want to make this bold...? This isn't proper Markdown.

File: README.md [hunk 3]
```diff
@@ -4,13 +4,22 @@ Arch Linux packaging for [ungoogled-chromium](//github.com/Eloston/ungoogled-chr
...
+
+[chaotic-aur](https://lonewolf.pedrohlc.com/chaotic-aur/) - Maintained by PedroHLC
```
Comment (Eloston) [inline README.md:18]: Nitpick: Probably better to make this a Markdown bulleted list.

File: README.md (hunk 2 above)
Comment (jstkdng) [reply README.md:16]: Weird, my markdown editor shows that as bold. huh

File: README.md (hunk 1 above)
Comment (jstkdng) [reply README.md:7]: understandable, will do

File: README.md (hunk 2 above)
Comment (jstkdng) [reply README.md]: just noticed the issue, it should show normally now.

File: README.md (hunk 3 above)
Comment (jstkdng) [reply README.md]: Done

**Comment by jstkdng** [comment]
@Eloston hope the changes are good now.

**Pull request merged**